operations/iuf/stages/management_nodes_rollout.md

```diff
@@ -1,19 +1,27 @@
 # `management-nodes-rollout`
 
-The `management-nodes-rollout` stage performs a controlled rebuild of the management NCNs in order to reboot them to a new image and configuration. A rebuild is a reboot operation that clears the persistent OverlayFS file system on
-that node. IUF will account for the necessary minimum number of critical software instances running on the nodes to ensure the `management-nodes-rollout` stage operates without impacting software availability.
-
-**`NOTE`** `management-nodes-rollout` currently does not rebuild management NCN storage nodes or `ncn-m001`. These nodes must be rebuilt using non-IUF methods described in the appropriate sections of the CSM documentation.
-
-- [Stage 1 - Ceph image upgrade](#../../upgrade/Stage_1.md) describes the rebuild process for NCN storage nodes
-- [Stage 2.3 - `ncn-m001` upgrade](#../../upgrade/Stage_2.md#stage-23---ncn-m001-upgrade) describes the rebuild process for `ncn-m001`
+The `management-nodes-rollout` stage performs a controlled update of management NCNs by configuring them with a new CFS configuration and/or rebuilding or upgrading them to a new image.
+A rebuild is a reboot operation that clears the persistent OverlayFS file system on that node.
+An upgrade is similar to a rebuild, but an upgrade does not wipe all information off of a node which is important
+specifically for NCN master and NCN storage nodes.
+An node image upgrade is only used during `management-nodes-rollout` if CSM is being upgraded.
```

Comment:
I think this will be misleading. On worker nodes, both COS and SHS currently add content to the base NCN worker image, and that modified image is what gets booted.  So CSM could remain at the same version, but we could take a patch for COS or SHS that results in an image "upgrade".   Maybe we should just remove this sentence?